include/pros/screen.h

```diff
@@ -0,0 +1,352 @@
+/**
+ * \file screen.h
+ *
+ * Brain screen display and touch functions.
+ *
+ * Contains user calls to the v5 screen for touching and displaying graphics.
+ *
+ * \copyright (c) 2017, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef _PROS_SCREEN_H_
+#define _PROS_SCREEN_H_
+
+#include <stdarg.h>   
+#include <stdbool.h>  
+#define _GNU_SOURCE
+#include <stdio.h>  
+#undef _GNU_SOURCE
+#include <stdint.h>
+
+#include "pros/colors.h"     // c color macros
+
+#ifdef __cplusplus
+extern "C" {
+namespace pros {
+#endif
+
+/**
+ * Different font sizes that can be used in printing text.
+ * Also can determine if the medium/large text is centered or not.
+ * 
+ *  small - Small text font size
```

Comment:
Our coding standards doc recommends using the Doxygen style enum comments, with these descriptors as `/** variable comment */` following the enum field like `E_TEXT_SMALL`